common/components/TaskCard/index.tsx

```diff
@@ -58,6 +59,8 @@ const TaskCard = ({ taskId, task }) => {
 
   const url = `https://maps.google.com/?q=${lat},${lng}`
   const [activeMarker, setActiveMarker] = useState(null)
+  const taskDeadline = moment(task?.deadline).format('YYYY-MM-DD')
+  const currentDate = moment().format('YYYY-MM-DD')
```

Comment:
We develop a website for Ukrainians, so I thought it would be better to change the date format to local.

Reply:
this date is not displayed